app/scripts/lib/setupSentry.js

```diff
@@ -666,9 +666,16 @@ export default function setupSentry({ release, getState }) {
   const startSession = async () => {
     const hub = Sentry.getCurrentHub?.();
     const options = hub.getClient?.().getOptions?.() ?? {};
-    if (hub && (await getSentryEnabled()) === true) {
+    const isMetaMetricsEnabled = (await getMetaMetricsEnabled()) === true;
+
+    if (hub && isMetaMetricsEnabled) {
```

Comment:
A benefit of leaving the enabled check inline is we only go to the effort or reading the state if `hub` is present?

Reply:
the reason behind this change was to having the code more readable. Since `getMetaMetricsEnabled` would be removed in the next pr I;'llbring back this inline change to cause less confusion for this PR 🙏🏻